### DISCUSSION TITLE
NRSV Bible Version of plugin

### DISCUSSION THREAD

**Discussion by DOK24** [tim-hub/obsidian-bible-reference#21] [Ideas]
This plugin has been so helpful. Was wondering if you had any plans for creating an NRSV version of the plugin? 

Thanks for your great work.

**Comment by tim-hub** [comment]
Hi @DOK24 NRSV is added in the 1.0.0 release

https://github.com/tim-hub/obsidian-bible-reference/releases/tag/1.0.2
